airbyte-integrations/bases/standard-source-test/src/main/java/io/airbyte/integrations/standardtest/source/AbstractSourceConnectorTest.java

```diff
@@ -125,11 +126,9 @@ public abstract class AbstractSourceConnectorTest {
   private ConnectorConfigUpdater mConnectorConfigUpdater;
 
   // This has to be using the protocol version of the platform in order to capture the arg
```

Comment:
I don't think this comment makes sense to keep anymore

Reply:
I think that meant to be AirbyteCatalog as it has to be using protocol version, but yeah definitely not used to capture the arg